pkgs/development/libraries/glm/default.nix

```
@@ -11,6 +16,14 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-F//+3L5Ozrw6s7t4LrcUmO7sN30ZSESdrPAYX57zgr8=";
   };
 
+  # https://github.com/g-truc/glm/pull/1055
+  # Fix more implicit-int-float-conversion warnings
+  # (https://github.com/g-truc/glm/pull/986 wasn't enough, and -Werror is used)
+  patches = [(fetchpatch {
+    url = "https://github.com/g-truc/glm/commit/bd9b5060bc3b9581090d44f15b4e236566ea86a6.patch";
```

I'd rather use the fork url, to make it clear it's not an upstream patch (yet).

```suggestion
    url = "https://github.com/kraj/glm/commit/bd9b5060bc3b9581090d44f15b4e236566ea86a6.patch";
```

And please send this as its own PR, for #144627

---

Will do

---

#147051